knowledge-base/common-remove-components-from-telerik-blazor-js.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Remove Components from telerik-blazor.js
+description: Learn 
+type: how-to
+page_title: How to Build the telerik-blazor.js File Without Some Components
+slug: common-kb-remove-components-from-telerik-blazor-js
+position: 
+tags: telerik, blazor
+ticketid: 1652093, 1632300
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>
+                UI for Blazor, <br />
+                PDF Viewert for Blazor, <br />
+                Spreadsheet for Blazor
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to build the `telerik-blazor.js` file without some of the components?
+* How to exclude components from the Telerik Blazor JSInterop file and rebuild `telerik-blazor.js`?
+* How to remove components from the client-side code of Telerik UI for Blazor?
+* How to remove the Spreadsheet component from `telerik-blazor.js` in order to achieve [strict Content Security Policy compliance]({%slug troubleshooting-csp%})?
+* How to remove the PDF Viewer component from `telerik-blazor.js` in order to avoid conflicts with other third-party tools that depend on **PDF.js**?
```

Comment:
```suggestion
* How to remove the PDF Viewer component from `telerik-blazor.js` in order to avoid conflicts with other third-party tools that depend on PDF.js?
```